### PR TITLE
Remove Duplicate information in `docs/cpp/cpp-ide.md`

### PR DESCRIPTION
**Pull request by 1chooo**
I have removed the duplicate content and kept the bulleted content above. Feel free to let me know if further adjustments are needed.

fixed: #7279, #7278

### CODE REVIEW

**Comment by 1chooo** [comment]
@ntrogh Could you please review this pull request? If there are no issues, could you merge it and allow contributions to the vscode-docs?

**Review by ntrogh — CHANGES_REQUESTED** [review]
Thanks for addressing these issues, @1chooo. Minor suggestion about the correct formatting for UI text.

File: docs/cpp/cpp-ide.md [hunk 1]
```diff
@@ -22,10 +22,9 @@ To provide the best editing experience, the C++ extension needs to know where to
 To specify additional include directories,
 
 1. Select an `#include` path that has no reference.
-1. Select the light bulb that appears and choose **Edit "includePath" setting**, which opens the C/C++ extension's Settings editor.
+1. Select the light bulb that appears and choose **`Edit "includePath" setting`**, which opens the C/C++ extension's Settings editor.
```
[the remark this answers is not in the document]
Comment (1chooo) [reply docs/cpp/cpp-ide.md]: I've already adjusted the order of the items in the list! Let me know if further adjustments are needed.

**Review by ntrogh — APPROVED** [review]
Thanks for the quick turnaround, @1chooo. LGTM! Appreciate your contribution to our docs.

**Pull request merged**